ui/app/utils/validators.js

```diff
@@ -62,4 +62,20 @@ export const isNonString = (value) => {
   }
 };
 
-export default { presence, length, number, containsWhiteSpace, endsInSlash, isNonString, hasWhitespace };
+export const WHITESPACE_WARNING = (item) =>
+  `${item} contains whitespace. If this is desired, you'll need to encode it with %20 in API requests.`;
```

Comment:
small nit: I'd push the responsibility of casing to this function so that we aren't relying on typing it with the correct casing every time we use it
```suggestion
  `${capitalize(item)} contains whitespace. If this is desired, you'll need to encode it with %20 in API requests.`;
```

Reply:
Good idea! will do